tests/system/action/meeting/test_import.py

```diff
@@ -40,7 +40,7 @@ def setUp(self) -> None:
             }
         )
 
-    def create_request_data(self, datapart: Dict[str, Any]) -> Dict[str, Any]:
+    def create_request_data(self, datapart: Dict[str, Any] = {}) -> Dict[str, Any]:
```

Comment:
Don't use a empty list or empty dict as a default, it is a can of worms.
If you add sth to that dict, it will continue to be in the dict, the next time you call it.
So it is *bad* style.

Reply:
done